### PR TITLE
Improve CQ2

### PR DESCRIPTION
**Pull request by iamcarbon**
A quick cleanup pass.

- Makes register sized classes structures
- Uses expression bodies in trival contexts
- Move properties above methods
- Small formatting improvements

### CODE REVIEW

**Comment by iamcarbon** [comment]
@heksesang Ready for review. If this looks OK, I'll take one final pass on formatting and then focus on adding test coverage.

**Comment by iamcarbon** [comment]
@heksesang This should cover the formatting / cleanup pass. If everything looks OK, I
l'll focus on adding test coverage next.

File: MatrixIO.IO/IO/ConstrainedStream.cs [hunk 1]
```diff
@@ -5,155 +5,145 @@
...
-        {
-            get 
-            { 
-                return Math.Min(_BaseStream.Length, _CurrentConstraint.End); 
-            }
-        }
+        public override long Length => Math.Min(_baseStream.Length, _currentConstraint.End);
 
-        public long _Count;
+        public long _count;
```
Comment (heksesang) [inline MatrixIO.IO/IO/ConstrainedStream.cs]: This probably shouldn't be publicly exposed like this.

Comment (iamcarbon) [reply MatrixIO.IO/IO/ConstrainedStream.cs]: Good catch. #resolved

**Comment by iamcarbon** [comment]
@heksesang Ready for review.

Code coverage coming next!

**Pull request merged**